src/OpenCV/ImgProc/ImgFiltering.hsc

```diff
@@ -296,6 +297,36 @@ blur size matIn =
   where ksize :: Size2i
         ksize = toSize2i size
 
+gaussianBlur
```

Comment:
I think it would be best in this case if you have a type saying it is either a kernel or `sigmaX` and `sigmaY`. So an `Either` or better a custom data type for this case.